### PR TITLE
Fix for mock store and :BindToClose()

### PR DESCRIPTION
**Pull request by FireBird702**

### CODE REVIEW

File: src/init.lua [hunk 1]
```diff
@@ -1023,30 +1023,28 @@ end
 
 local saveLoop
 
-if RunService:IsStudio() then
-	game:BindToClose(function()
-		Store.ServiceDone = true
-		Keep.ServiceDone = true
+game:BindToClose(function()
```
Comment (noahrepublic) [inline src/init.lua]: Seems I forgot to add back a "not" when testing. It was intended to not BindToClose in studio so it does not cause the servers to stay up. And, playerremoving seems enough for this. If could just add that would be great

Comment (noahrepublic) [reply src/init.lua]: And sorry for pushing something, but resolve the small conflict. just one-line removal

Comment (FireBird702) [reply src/init.lua]: > Seems I forgot to add back a "not" when testing. It was intended to not BindToClose in studio so it does not cause the servers to stay up. And, playerremoving seems enough for this. If could just add that would be great

Why would servers stay up?
Also if you :LoadKeep() in studio that is not a player keep and not release it when studio session is closed wouldn't it make the keep session locked forever?

**Comment by noahrepublic** [comment]
I can't seem to fix merge requests from here, once you fix that I can merge

**Comment by noahrepublic** [comment]
I think you're write about the BindToClose in studio. I don't wanna bother you to change, I will merge and change with my commits

**Review by noahrepublic — APPROVED** [review]
Looks good, thanks

**Pull request merged**